apps/user-service/src/auth/reset-password.service.ts

```diff
@@ -0,0 +1,81 @@
+import { Injectable, NotFoundException, BadRequestException, Logger } from "@nestjs/common";
+import bcrypt from 'bcryptjs';
+import { JwtService } from '@nestjs/jwt';
+import { User } from '@terramatch-microservices/database/entities';
+import { EmailService } from "@terramatch-microservices/common/email/email.service";
+import { LocalizationService } from "@terramatch-microservices/common/localization/localization.service";
+
+@Injectable()
+export class ResetPasswordService {
+  private readonly logger: Logger;
+
+  constructor(
+    private readonly jwtService: JwtService,
+    private readonly emailService: EmailService,
+    private readonly localizationService: LocalizationService
+  ) {
+    this.logger = new Logger(ResetPasswordService.name);
+  }
+
+  async sendResetPasswordEmail(emailAddress: string, callbackUrl: string) {
+    const user = await User.findOne({ where: { emailAddress } });
```

Comment:
It looks like you're only using id, uuid and locale from the user so I'd limit this query to those attributes:
```
const user = await User.findOne({ where: { emailAddress }, attributes: ["id", "uuid", "locale"]);
```